exporters/committime/collector_base.py

```diff
@@ -67,6 +67,8 @@ class AbstractCommitCollector(pelorus.AbstractPelorusExporter):
 
     namespaces: set[str] = field(factory=set, converter=comma_separated(set))
 
+    prod_label: str = field(default=pelorus.DEFAULT_PROD_LABEL)
```

Comment:
this adds `PROD_LABEL` to git and image(stream) committime exporter, don't? I think it is wrong, should be a config only for new exporter

Reply:
I've added this to collector_base as I think it's important to rework the commit time exporter(s) to use:

```
get_and_log_namespaces()
```

From:
```
provider_common.openshift
```

But I did not want to make such change in this PR as it's not relevant to the skopeo work, however though the PROD_LABEL natural  place is on the base abstract class.

Reply:
This one should be replaced with get_and_log_namespaces() which does support PROD_LABEL to be consistent with deploytime exporter as well:

https://github.com/dora-metrics/pelorus/blob/master/exporters/committime/collector_base.py#L135-L146

Reply:
I am fine with this only if we add NAMESPACES and PROD_LABEL options to all committime exporters